### PR TITLE
Remove https-everywhere-checker message (2nd run)

### PR DESCRIPTION
**Pull request by cschanaj**
As per https://github.com/EFForg/https-everywhere/pull/11274#issuecomment-318867682

15 of 21 files failed `fetch-test` ... 

```bash
$ grep ERROR 189759654.log | sed 's/ERROR \([^:]*\).*/\1/g' | sort | uniq  
src/chrome/content/rules/7_Springs.com.xml
src/chrome/content/rules/Alfa-Media-Partner.xml
src/chrome/content/rules/Auto_Rims_and_Accessories.xml
src/chrome/content/rules/Brixwork.com.xml
src/chrome/content/rules/BTCZone.xml
src/chrome/content/rules/Covered-California.xml
src/chrome/content/rules/E-Estonia.com.xml
src/chrome/content/rules/Laup.xml
src/chrome/content/rules/Matkahuolto.xml
src/chrome/content/rules/McClatchy_Interactive.com.xml
src/chrome/content/rules/Mirror_Bingo.com-falsemixed.xml
src/chrome/content/rules/Mstar.com.xml
src/chrome/content/rules/Obedovat.sk.xml
src/chrome/content/rules/Packetwerk.com.xml
src/chrome/content/rules/Pan-Islamic_Malaysian_Party.xml
```

### CODE REVIEW

**Comment by cschanaj** [comment]
All files changed pass `rules` test as of 36ad514c28477331e38e80aac89cc8031af3750a. I am going to disable the files listed above.

File: src/chrome/content/rules/Tandlakarforbundet.se.xml [hunk 1]
```diff
@@ -1,7 +1,3 @@
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://tandlakarforbundet.se/ => https://www.tandlakarforbundet.se/: (7, 'Failed to connect to www.tandlakarforbundet.se port 443: Connection refused')
-Fetch error: http://www.tandlakarforbundet.se/ => https://www.tandlakarforbundet.se/: (7, 'Failed to connect to www.tandlakarforbundet.se port 443: Connection refused') tandlakarforbundet is a swedish labour union -->
 <ruleset name="Tandlakarforbundet.se">
```
Comment (Bisaloo) [inline src/chrome/content/rules/Tandlakarforbundet.se.xml]: It doesn't really matter here but just so you know, your script sometimes removes comments that weren't added by travis

Comment (cschanaj) [inline src/chrome/content/rules/Tandlakarforbundet.se.xml]: This is not a bug but an edge case for _https-everywhere-checker_.  (see [`check_rules.py#L218`](https://github.com/EFForg/https-everywhere/blob/master/test/rules/src/https_everywhere_checker/check_rules.py#L218)) 

Since the original comment did not start with a newline (923fd05e), neither do _https-everywhere-checker_ suffixed its message with newline `\n\n` (b69a6d7a in 2015). The original comment and checker's message ended up in the same line. 

Since this change does not matter here I am not going to fix it. I will pay more attention to ruleset like this in later PR. 

Thanks!!

**Comment by J0WI** [comment]
So you removed the https-everywhere-checker message and just whitelist the rules that fail our tests?
I think we should check why they fail before we just ignore the errors.

**Comment by cschanaj** [comment]
@J0WI this is fixed. thanks!

P.S. I will open a separate issue for those problematic files later.
